src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/spec/SpecGerritTriggerHudsonTest.java

```diff
@@ -552,6 +553,7 @@ public void testBuildLatestPatchsetOnlyNonRelatedChangeDifferentBranchCannotAbor
 
         GerritTrigger trigger = project.getTrigger(GerritTrigger.class);
         trigger.setSilentMode(true);
+        trigger.setSilentLevel(SilentLevel.ALL);
```

Comment:
`setSilentMode` should be no more and replaced by `trigger.setSilentLevel(SilentLevel.ALL)` If I've understood your logic correctly.